### PR TITLE
Change default install profile to sitenow so it runs in CI.

### PR DESCRIPTION
**Pull request by richardbporter**
Currently, the standard profile is being installed which isn't testing configuration integrity. This just changes the default profile to sitenow so that it is used during CI.

### CODE REVIEW

**Comment by richardbporter** [comment]
I need to change a few more things around before this will work...

File: blt/blt.yml [hunk 1]
```diff
@@ -54,6 +54,12 @@ command-hooks:
   post-deploy-build:
     dir: ${repo.root}
     command: './vendor/bin/blt uiowa:post:build'
+cm:
+  core:
+    install_from_config: true
+    dirs:
+      sync:
+        path: profiles/custom/sitenow/config/sync
```
Comment (richardbporter) [inline blt/blt.yml:62]: This should be removed after #1477.

**Comment by richardbporter** [comment]
This is a quick fix with #1477 being the long term goal. The default site might need to be re-installed when deploying this. I can handle that.

**Review by pyrello — APPROVED** [review]
Not tested, but reviewed the output from CI, which is what this should affect.

**Pull request merged**